ciao-image/datastore/store.go

```diff
@@ -178,11 +178,13 @@ func (s *ImageStore) UploadImage(ID string, body io.Reader) error {
 	if s.rawDs != nil {
 		err = s.rawDs.Write(ID, body)
 		if err != nil {
+			database.Logger.Errorf("Could not write image: %v", err)
```

Comment:
I think we should have our own Logger in ``ciao-image``. According to what is defined [here](https://github.com/01org/ciao/tree/master/clogger) each component needs to implement its Logger. For this ``ciao-image`` we could reuse the already glog backend interface that clogger is proposing at https://github.com/01org/ciao/tree/master/clogger/gloginterface

Here are some examples how this clogger is being implemented:
- [database package](https://github.com/01org/ciao/blob/master/database/logger.go)
- [networking/libsnnet package](https://github.com/01org/ciao/blob/master/networking/libsnnet/logger.go)

As far I can see those are the packages that are using this already. @sboeuf could you please comment on this? I saw you already used that clogger in those 2 packages. For ``ciao-image``, we should add a new ``logger.go``, right?